zk.js/src/build-balance.ts

```diff
@@ -38,9 +38,9 @@ export class TokenUtxoBalance {
   tokenData: TokenData;
   totalBalanceSpl: BN;
   totalBalanceSol: BN;
-  utxos: Map<string, Utxo>; // commitment hash as key
-  committedUtxos: Map<string, Utxo>; // utxos which are
-  spentUtxos: Map<string, Utxo>; // ordered for slot spent - maybe this should just be an UserIndexedTransaction
+  utxos: Map<string, Utxo | ProgramUtxo<PlaceHolderTData>>; // commitment hash as key
+  committedUtxos: Map<string, Utxo | ProgramUtxo<PlaceHolderTData>>; // utxos which are
```

Comment:
no need for comitted utxos

Reply:
good catch, will be redundant changes in follow up pr